### DISCUSSION TITLE
Wrong disk size info with Linstore

### DISCUSSION THREAD

**Discussion by leo79901** [apache/cloudstack#10142] [Q&A] [answered]
#### In the Linstore, I have created a pool with LVMTHIN and used it in ACS.
![image](https://github.com/user-attachments/assets/fe170a83-139e-4d37-a7c0-47c702c3836c)

#### While I created a VM with thin provisioning, I found that it was fully used.
![image](https://github.com/user-attachments/assets/039e8953-90e0-4ae7-b9a7-e8e314021458)

#### But in Linstore, it was with thin provisioning.
![image](https://github.com/user-attachments/assets/b49f6b07-30e9-4d47-aad9-b8018188df44)

#### Perhaps it is a bug related to the display?

**Comment by DaanHoogland** [comment]
@leo79901 , do you have a resolution to this query yet?

**Reply by leo79901** [reply]
Sorry, not yet.

**Reply by DaanHoogland** [reply]
@rp- , can you help here?

**Reply by rp-** [reply, marked as answer]
Wasn't aware these numbers come from the volumestats.
And that is not implement (yet) with Linstor, I'll commit a PR in 1 or 2 weeks.

**Reply by rp-** [reply]
https://github.com/apache/cloudstack/pull/10850
